wasmsdk/cache.go

```diff
@@ -18,16 +19,23 @@ var (
 	cachedAllocations, _ = lru.New[string, *cachedAllocation](100)
 )
 
-func getAllocation(allocationID string) (*sdk.Allocation, error) {
-	it, ok := cachedAllocations.Get(allocationID)
+func getAllocation(allocationId string, refresh bool) (*sdk.Allocation, error) {
```

Comment:
The refresh may confuse users and may difficult to choose whether they should do refresh or not, or in most of the case, they would simply chose to refresh. So why not remove the param and just do the refresh logic.

Reply:
removed `refresh`, and added `reloadAllocation`